.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: Test application (e2e)
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Run NPM Test (e2e)
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [13.x]
+        mongodb-version: [4.2]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+  
+      - name: Start MongoDB
+        uses: supercharge/mongodb-github-action@1.3.0
+        with:
+          mongodb-version: ${{ matrix.mongodb-version }}
+
+      - name: NPM Test
+        working-directory: ./server
+        env:
+          DB_SERVER: localhost
+          DB_USER: development
+          DB_PASSWORD: development
+          DB_NAME: development
+          DB_PORT: 27017
```

Comment:
Part of the current issue might be related to these... This is a DB instance that only lives for the duration of the tests run, right? The tests are expecting that hardcoded test DB name I was asking about earlier 😂 

Best path forward is probably to parameterize the test DB info and get it into secrets?

Reply:
No need anymore. As you said these parameters don’t make sense since is a short-lived DB (neither the plugin has support for this), I just change for a non-auth and now the tests are working properly :)

Reply:
Awesome! That's way better 🙌